changelog/README.md

```diff
@@ -4,6 +4,11 @@
 * [Get all order items](../operations/orderitems.md#get-all-order-items):
   * Extended request object with `AccountIds` filtering parameter.
 
+## 3rd March 2025
+* [Get all cashier transactions)](../operations/cashiertransactions.md#get-all-cashier-transactions):
+  * Incorrectly named filtering parameter `UpdatedUtc` changed to `CreatedUtc`. Documentation only, no changes to API functionality.
+  * Extended request object with `ActualEndUtc` filtering parameter.

```

Comment:
question: Are you sure this changelog entry belongs here? Could this be a result of bad merge? The entry is already published: https://github.com/MewsSystems/gitbook-connector-api/blob/master/changelog/README.md#21st-february-2025